package.json

```diff
@@ -48,6 +48,10 @@
     "rimraf": "^2.5.3"
   },
   "dependencies": {
+    "lodash.identity": "^3.0.0",
+    "lodash.isfunction": "^3.0.8",
+    "lodash.isnil": "^4.0.0",
+    "lodash.issymbol": "^4.0.1",
```

Comment:
Using `lodash` is fine, but I'd suggest to just go with `lodash` as dependency and use e.g. `import identity from 'lodash/identity'` instead. It has same effect but without polluting the `package.json` and we have access to all `lodash` libs if needed without installing new deps.